src/bus/index.js

```diff
@@ -12,13 +15,73 @@ class Bus {
    */
   constructor(sdk, request) {
     const baseUrl = `${sdk.config.audiences.bus.host}`;
-
+    const baseSocketUrl = `${sdk.config.audiences.bus.socket}`;
+    this._baseSocketUrl = baseSocketUrl;
     this._baseUrl = baseUrl;
     this._request = request;
     this._sdk = sdk;
+    this._sockets = {};
 
     this.channels = new Channels(sdk, request, baseUrl);
   }
+
+  /**
+   * Connects to the message bus via websocket.
+   * If a connection already exists for that organization id, the connection is returned, otherwise a new connection is created.
```

Comment:
Might want to add on and say `otherwise a new connection is created and returned.`

Reply:
👍